### PR TITLE
Always save completed file reviews

### PR DESCRIPTION
**Pull request by salbertson**
We noticed that new file review records are being created without `completed_at` values. This should avoid that and not save any file reviews without `completed_at`.

### CODE REVIEW

**Comment by croaky** [comment]
Are SCSS, CoffeeScript, and JavaScript all set with the completed bit? Thought I'd see specs for those like Ruby and Haml.

Also, those Ruby style specs are really noisy, seem like they're testing Rubocop. Do we need them or is it better to delete them for less maintenance?

File: app/models/style_guide/base.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@ def enabled?
       repo_config.enabled_for?(name)
     end
 
-    def file_included?(_file)
+    def file_included?(*)
```
Comment (gylaz) [inline app/models/style_guide/base.rb]: Should we remove this from base altogether? That way a new linter doesn't forget to implement it?

**Comment by gylaz** [comment]
:+1:

**Comment by salbertson** [comment]
@croaky, yup, all the style guide classes should response to `#file_review` with completed file reviews. I went through the style guide specs, looks like we covered them all.

Yea, there are a lot of tests covering RuboCop cases, but mostly because we are using RuboCop in an abnormal way.

File: app/models/style_guide/base.rb (hunk 1 above)
Comment (salbertson) [inline app/models/style_guide/base.rb]: What do you think about raising? Check out the latest commit.

**Pull request merged**